packages/itwinui-react/src/core/Alert/Alert.tsx

```diff
@@ -136,7 +136,7 @@ const AlertAction = React.forwardRef((props, ref) => {
   return (
     <ButtonBase
       as={(!!props.href ? 'a' : 'button') as 'a'}
-      className={cx('iui-alert-link', className)}
+      className={cx('iui-anchor iui-alert-link', className)}
```

Comment:
Should `.iui-anchor` only be applied when the `as` is an `<a>`?

Reply:
The type on that `forwardRef` indicates that the `AlertAction` will always be an `<a>`, which is a little confusing with the `as` potentially being a `<button>`. I will investigate this and see if there are any issues with only applying the `.iui-anchor` class when `as` is `<a>`.

Reply:
I can confirm that the visual tests expect the styles that we provide with the `.iui-anchor` class, even when the element is a `<button>`. It therefore makes sense to apply the class to the `AlertAction`, regardless of the `as` parameter.